src/rules/flake8_return/visitor.rs

```diff
@@ -15,6 +15,7 @@ pub struct Stack<'a> {
     pub assigns: FxHashMap<&'a str, Vec<Location>>,
     pub loops: Vec<(Location, Location)>,
     pub tries: Vec<(Location, Location)>,
+    pub assign_values: FxHashMap<&'a str, rustpython_ast::Located<ExprKind>>,
```

Comment:
As a tip, we never need to use `Located<T>`. Instead, you can do `Expr` (which is equivalent to `Located<ExprKind>`).

Reply:
Thanks for the tip!